web/src/main/java/se/inera/intyg/intygstjanst/web/service/CitizenCertificateFilterService.java

```diff
@@ -0,0 +1,20 @@
+package se.inera.intyg.intygstjanst.web.service;
+
+import se.inera.intyg.intygstjanst.web.integration.citizen.CitizenCertificateStatusTypeDTO;
+import se.inera.intyg.intygstjanst.web.service.dto.citizen.CitizenCertificateDTO;
+
+import java.util.List;
+
+public interface CitizenCertificateFilterService {
+    boolean filterOnYears(CitizenCertificateDTO certificate, List<String> includedYears);
+
+    boolean filterOnSentStatus(CitizenCertificateDTO certificate, List<CitizenCertificateStatusTypeDTO> statuses);
+
+    boolean filterOnUnits(CitizenCertificateDTO certificate, List<String> unitIds);
+
+    boolean filterOnCertificateTypes(CitizenCertificateDTO certificate, List<String> certificateTypes);
+
+    boolean filter(CitizenCertificateDTO certificate, List<String> includedYears, List<String> unitIds,
```

Comment:
Is this bad practice to add one filter method which does all filtering

Reply:
Not really, but maybe the API should just have a filter(....) method and filtering on specific attributes are internal workings of the Filter-class.